dbt_platform_helper/templates/pipelines/codebase/overrides/stack.ts

```diff
@@ -403,7 +406,10 @@ export class TransformedStack extends cdk.Stack {
         const buildProjectPolicy = this.template.getResource("BuildProjectPolicy") as cdk.aws_iam.CfnPolicy;
         (buildProjectPolicy.policyDocument.Statement as Array<any>).push({
             Effect: 'Allow',
-            Action: ['codestar-connections:UseConnection'],
+            Action: [
+                'codestar-connections:GetConnectionToken',
```

Comment:
I'm not sure why this has become needed, but it has, and is likely to affect other applications too.